src/Sdk/NewsroomPrivacyRequests/NewsroomPrivacyRequests.ts

```diff
@@ -0,0 +1,47 @@
+import { Newsroom, PrivacyRequest } from '../../types';
+
+import DeferredJobsApiClient from '../DeferredJobsApiClient';
+import routing from '../routing';
+import {
+    DeletePrivacyRequestCreateRequest,
+    ExportPrivacyRequestCreateRequest,
+    CorrectPrivacyRequestCreateRequest,
+    OtherPrivacyRequestCreateRequest,
+} from './types';
+
+export default class NewsroomPrivacyRequests {
+    private readonly apiClient: DeferredJobsApiClient;
+
+    constructor({ apiClient }: { apiClient: DeferredJobsApiClient }) {
+        this.apiClient = apiClient;
+    }
+
+    public async create(
+        newsroomId: Newsroom['uuid'],
+        payload:
+            | DeletePrivacyRequestCreateRequest
+            | ExportPrivacyRequestCreateRequest
+            | CorrectPrivacyRequestCreateRequest
+            | OtherPrivacyRequestCreateRequest,
+    ): Promise<PrivacyRequest> {
+        const url = routing.newsroomPrivacyRequestsUrl.replace(':newsroom_id', String(newsroomId));
+        const { privacy_request } = await this.apiClient.post<{ privacy_request: PrivacyRequest }>(
+            url,
+            {
+                payload,
+            },
+        );
+        return privacy_request;
+    }
+
+    public async confirm(
```

Comment:
We're not gonna actually use this method, right?

Currently the data request is confirmed by clicking a link in the email as far as I know and then you're redirected to the newsroom's "confirmation successful" page.

Does this approach stay the same? We don't have such screen in Figma, should we add it?

Reply:
I think this should be handled similar as for subscribing. We are sending email from privacy portal, not from backend.

So the steps (from privacy portal) would be:
1. Create privacy request
2. Send confirmation email
3. When confirmation link is clicked, send `confirm` request to API and display success message

Does this make sense or you have a different suggestion?

Reply:
Yeah, it makes sense. So I'll have to add the confirmation page as well then. This can be merged 👍